canon-docs/src/utils/changelog.ts

```diff
@@ -0,0 +1,376 @@
+export type Component =
+  | 'avatar'
+  | 'box'
+  | 'button'
+  | 'heading'
+  | 'text'
+  | 'icon-button'
+  | 'icon'
+  | 'tabs'
+  | 'menu'
+  | 'textfield'
+  | 'datatable'
+  | 'select'
+  | 'collapsible'
+  | 'checkbox'
+  | 'container'
+  | 'link'
+  | 'tooltip'
+  | 'scrollarea'
+  | 'flex';
+
+export interface ChangelogProps {
+  components: Component[];
+  description: string;
+  version: '0.1.0' | '0.2.0' | '0.3.0' | '0.4.0' | '0.5.0' | '0.6.0';
```

Comment:
🤔, maybe `${string}.${string}.${string}` instead if you want to keep it a bit strict?

Reply:
You're right. This will look better.